common/src/main/java/com/wynntils/core/webapi/WebManager.java

```diff
@@ -67,6 +72,8 @@ public class WebManager {
 
     private static @Nullable WynntilsAccount account = null;
 
+    private static @Nullable MapProfile map = null;
```

Comment:
Ditch the `@Nullable`. They are just making the code cluttered.

Reply:
Will do.